### PR TITLE
Fix Blackduck warning

### PR DESCRIPTION
**Pull request by anton-dahlin**
Fixed with npm audit

### CODE REVIEW

**Review by cbardini — APPROVED** [review]

**Comment by derekendres** [comment]
version/history

File: HISTORY.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Version History
 
-## 1.1.9 / 2022-01-27
+## 1.1.10 / 2022-02-28
 
-- Updated for AVEVA Data Hub
+- Updated dependencies
```
Comment (kelseyduffy) [inline HISTORY.md:5]: it looks like we lost the 1.1.9 entry when adding the 1.1.10 entry

**Review by kelseyduffy — CHANGES_REQUESTED** [review]
history file 1.1.9 entry got overwritten

File: HISTORY.md (hunk 1 above)
Comment (anton-dahlin) [reply HISTORY.md]: thank you!

**Review by kelseyduffy — APPROVED** [review]

**Pull request merged**